notebook/gateway/managers.py

```diff
@@ -401,7 +401,9 @@ async def start_kernel(self, kernel_id=None, path=None, **kwargs):
 
             json_body = json_encode({'name': kernel_name, 'env': kernel_env})
 
-            response = await gateway_request(kernel_url, method='POST', body=json_body)
+            response = await gateway_request(
+                kernel_url, method='POST', headers={'Content-Type': 'application/json'}, body=json_body
```

Comment:
Since `headers` can be set via configuration, `gateway_request()` (or more likely `load_connection_args()`) needs to now handle the case where a given keyword already exists in the static arguments and merge accordingly.  Up until now, that _key-level_ merge wasn't necessary.

Reply:
Done.